processingchain/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/Workflow.groovy

```diff
@@ -131,7 +131,7 @@ def BDTOPO_V2() {
                              }
                              else{
                                  def tablesToSave = geoIndicators.getResults().collect {
-                                     if (datasource.hasTable(it)){
+                                     if (it.value != null){
```

Comment:
You can replace `if(it.value != null)` by `if(it.value)` (for more info : [Groovy truth](https://groovy-lang.org/semantics.html#Groovy-Truth))

Reply:
done